running-a-nats-service/configuration/README.md

```diff
@@ -151,30 +151,32 @@ By default, the JetStream subsystem will store data in the /tmp directory, but y
 
 | Property | Description | Default | Version |
 | :--- | :--- | :--- | :--- |
-| `store_dir` | Directory to use for JetStream storage. | `/tmp/nats/jetstream` | 2.2+ |
-| `max_memory_store` | Maximum size of the 'memory' storage | 75% of available memory | 2.2+ |
-| `max_file_store` | Maximum size of the 'file' storage | Up to 1TB if available | 2.2+ |
-| `max_outstanding_catchup` | Max in-flight bytes for stream catch-up | 32MB | 2.9+ |
+| `store_dir` | Directory to use for JetStream storage. | `/tmp/nats/jetstream` | 2.2.0 |
+| `max_memory_store` | Maximum size of the 'memory' storage | 75% of available memory | 2.2.0 |
+| `max_file_store` | Maximum size of the 'file' storage | Up to 1TB if available | 2.2.0 |
+| `cipher` | Set to enable storage-level [encryption at rest][encryption]. Choose either `chachapoly` or `aes`. | (not set) | 2.3.0 |
+| `key` | The encryption key to use when [encryption at rest][encryption] is enabled. Choose either `chachapoly` or `aes`. | (not set) | 2.3.0 |
```

Comment:
@bruth copy-paste error on description of `key`.

Reply:
@bruth the encryption key is HMAC256 on startup.   Technically, `key` could be any length of characters, but would recommend minimum 32 bytes in length.  Anything greater than 64 from my understanding gets pre-hash/transformed to a 64-byte in the HMAC256 algorithm.